pkg/web/handlers.go

```diff
@@ -28,7 +28,7 @@ func RecoveryHandler(w http.ResponseWriter, r *http.Request, err interface{}) {
 	switch internalErr := err.(type) {
 	case *errors.Error:
 		log.WithFields(log.Fields{"code": internalErr.Code, "error": internalErr.Error()}).
-			Warning("Internal error hadled")
+			Info("Internal error handled")
```

Comment:
This is handled error that we `panic()`, so it is logged with proper level before throwing. `Error` is something that really requires attention.